2200071/문제3.py

```diff
@@ -0,0 +1,9 @@
+T = int(input())
+for i in range(1, T + 1):
+    N = int(input())
+    a = list(map(int,input().split()))
+    b = sum(a)/N
+    for number in a:
+        if number <= b:    
+            number = []
+            print(f'#{i} {len(number)}')
```

Comment:
평균을 `sum` 함수를 활용해 잘 구해주셨네요! 👍
답 구하는 과정만 조금 고치면 될 것 같습니다.
```suggestion
    b = sum(a)/N
    result = []       # 평균보다 작은 점수들을 저장할 리스트
    for number in a:
        if number <= b:                      # 평균보다 작다면?
            result.append(number)     # 리스트에 추가해주자
    print(f'#{i} {len(result)}')           # 답은 리스트의 길이가 된다!
```